plugins/github-pull-requests-board/src/components/TeamPullRequestsBoard/TeamPullRequestsBoard.tsx

```diff
@@ -0,0 +1,120 @@
+import { Wrapper } from '../Wrapper';
+import { PullRequestCard } from '../PullRequestCard';
+import { usePullRequestsByTeam } from '../../hooks/usePullRequestsByTeam';
+import { PRCardFormating } from '../../utils/types';
+import { DraftPrIcon } from '../icons/DraftPr'
+import { useUserRepositories } from '../../hooks/useUserRepositories';
+
+const TeamPullRequestsBoard: FunctionComponent = () => {
+  const [infoCardFormat, setInfoCardFormat] = useState<PRCardFormating[]>([]);
+  const { repositories } = useUserRepositories();
+  const { loading, pullRequests, refreshPullRequests } = usePullRequestsByTeam(repositories);
+
+  const header = (
+    <InfoCardHeader onRefresh={refreshPullRequests}>
+      <PullRequestBoardOptions
+        onClickOption={(newFormats) => setInfoCardFormat(newFormats)}
+        value={infoCardFormat}
+        options={[
+          {
+            icon: <DraftPrIcon />,
+            value: 'draft',
+            ariaLabel: 'Show draft PRs'
+          },
+          {
+            icon: <FullscreenIcon />,
+            value: 'fullscreen',
+            ariaLabel: 'Info card is set to fullscreen'
+          }
+        ]}
+      />
+    </InfoCardHeader>
+  );
+
+  const getContent = () => {
+    if (loading) {
+      return <Progress />;
+    }
+
+    return (
+      <Grid container spacing={2}>
+        {pullRequests.length ? (
+          pullRequests.map(({ title: columnTitle, content }) => (
+            <Wrapper
+              key={columnTitle}
+              fullscreen={infoCardFormat.includes('fullscreen')}
+            >
+              <Typography variant="overline">
+                {columnTitle}
+              </Typography>
+              {content.map(({
+                id,
+                title,
+                createdAt,
+                lastEditedAt,
+                author,
+                url,
+                latestReviews,
+                repository,
+                isDraft
+              }, index) => (
+                isDraft ? (infoCardFormat.includes('draft') === isDraft) &&
+                  <PullRequestCard
+                    key={`pull-request-${id}-${index}`}
+                    title={title}
+                    createdAt={createdAt}
+                    updatedAt={lastEditedAt}
+                    author={author}
+                    url={url}
+                    reviews={latestReviews.nodes}
+                    repositoryName={repository.name}
+                    isDraft={isDraft}
+                  />
+                  : <PullRequestCard
+                    key={`pull-request-${id}-${index}`}
+                    title={title}
+                    createdAt={createdAt}
+                    updatedAt={lastEditedAt}
+                    author={author}
+                    url={url}
+                    reviews={latestReviews.nodes}
+                    repositoryName={repository.name}
+                    isDraft={isDraft}
+                  />
```

Comment:
Aren't these two <PullRequestCard .../> components exactly the same here ? Why do we need the ternary operator?